youtube_dl/extractor/ceskatelevize.py

```diff
@@ -197,7 +268,10 @@ def _real_extract(self, url):
         for e in entries:
             self._sort_formats(e['formats'])
 
-        return self.playlist_result(entries, playlist_id, playlist_title, playlist_description)
+        if len(entries) > 1:
+            return self.playlist_result(entries, playlist_id, playlist_title, playlist_description)
+        else:
+            return entries[0]
```

Comment:
Don't.

Reply:
Can you explain why? This doesn't seem to serve any real purpose, returning a playlist with one nested item. At the same time, it makes it a lot harder/impossible to combine videos on eg. a news page into a playlist, because this would have 3 levels, which
 - is impossible to test
 - confuses other tools while parsing
I can mostly fix the other little things, but this one seems crucial.